build.properties

```diff
@@ -2,7 +2,7 @@
 
 
 # Project specific information
-package=com.nutrons.steamworks
+package=com.nutrons.stronghold
```

Comment:
Keep this as steamworks.  This is all test code for our actual robot at this point, and we don't want to have package issues if people copy and paste.